.gitignore

```diff
@@ -1,10 +1,8 @@
-# Default ignored files
-/shelf/
-/workspace.xml
-# Datasource local storage ignored files
-/dataSources/
-/dataSources.local.xml
-# Editor-based HTTP Client requests
-/httpRequests/
```

Comment:
Maybe we should leave these because the folder structure is different on other machines

Reply:
Solved in https://github.com/NHristovski/ServerHelper-ClientApp/pull/1/commits/20c1f00428f8edfa8feef7a5b8d3a11edae87789.
However, if `.gitignore` is not in the top directory, then it won't work anyway.